BotModules/ScreechBot/index.js

```diff
@@ -1,25 +1,18 @@
-const {
-  LOOPDELAY, // Delays the watch loop on the server by milliseconds to reduce server overhead. larger delay is less responsive.
-  GREETING_DELAY // Give some time before the greeting is played to allow the user to connect to the voice chat successfully.
-} = require('./config.json');
+// Delays the watch loop on the server by milliseconds to reduce server overhead. larger delay is less responsive.
+const LOOPDELAY = process.env.SCREECHBOT_LOOPDELAY ? process.env.SCREECHBOT_LOOPDELAY : 100;
+// Give some time before the greeting is played to allow the user to connect to the voice chat successfully.
+const GREETING_DELAY = process.env.SCREECHBOT_GREETING_DELAY ? process.env.SCREECHBOT_GREETING_DELAY : 2000;
 
 const EventEmitter = require('events');
 const path = require('path');
+const Commander = require('../CommanderInChief/commander-in-chief');
```

Comment:
Instead of having all bots require the commander can we set up an interface that all bots export a GetCommands() function?

Then Meggabot would would use commander.register(somebot.getCommands); for each bot. 

I feel like that would help decouple our bots from eachother.

Reply:
Agreed. updated in commit 8956031dab6a1252d5aa0241e88522f6c953fe77